dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp

```diff
@@ -767,22 +768,29 @@
                     fileNameField?.setValue(newFileName);
                     dijit.byId("confirmReplaceNameDialog-<%=field.getVelocityVarName()%>").hide();
                 }
-                
+
                 function closeConfirmReplaceName(){
                     dijit.byId("confirmReplaceNameDialog-<%=field.getVelocityVarName()%>").hide();
                 }
             </script>
             <script>
                 // Create a new scope so that variables defined here can have the same name without being overwritten.
-                (function autoexecute() {
+                function setBinaryFieldInputs() {
```

Comment:
What happens if you have more than one Binary field in the same `content type`?

Reply:
I will check this. I think it will fail

Reply:
It did fail, but I already got the fix. Outstanding catch